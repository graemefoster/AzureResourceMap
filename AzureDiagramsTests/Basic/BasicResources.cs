using Shouldly;

namespace AzureDiagramsTests.Basic;

public class BasicResources
{
    [Fact]
    public async Task SingleStorageAccount()
    {
        var resources = TestResourcesObjectMother
            .WithPublicAccessibleStorageAccount();

        var diagram = await AzureDiagramGenerator.DrawIoDiagramGenerator.DrawDiagram(
            resources.ToArray(),
            false, 
            false, 
            false, 
            false,
            false);
        
        diagram.ShouldMatchApproved();
    }
}