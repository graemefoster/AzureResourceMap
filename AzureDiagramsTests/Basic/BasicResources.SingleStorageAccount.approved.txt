<mxGraphModel>
	<root>
		<mxCell id="0" />
		<mxCell id="1" parent="0" />
<mxCell id="469266b2-b714-ab6b-0de0-6c2c5d9d6328" value="storage123" style="html=1;image;image=img/lib/azure2/storage/Storage_Accounts.svg;fontSize=12;labelPosition=bottom" vertex="1" parent="1">
    <mxGeometry x="-25" y="500" width="50" height="50" 
    as="geometry" />
</mxCell>


	</root>
</mxGraphModel>